packages/ember-glimmer/tests/integration/syntax/with-test.js

```diff
@@ -174,7 +175,7 @@ moduleFor('Syntax test: {{#with as}}', class extends TogglingSyntaxConditionalsT
 
```

Comment:
can you change line 170 to set `proxyThing.content`? We should not be setting `isTruthy` directly (and I guess line 174 doesn't make sense anymore)